### PR TITLE
Create branch page for ja

### PR DESCRIPTION
**Pull request by hsbt**
https://www.ruby-lang.org/en/downloads/branches/ page is only provided en page. I added same page to `ja` translation.

### CODE REVIEW

File: ja/downloads/branches/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: "Ruby のメンテナンスバージョン"
```
Comment (sorah) [inline ja/downloads/branches/index.md:3]:
```suggestion
title: "Ruby ブランチごとのメンテナンス状況"
```

File: ja/downloads/branches/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+lang: ja
+---
+
+このページでは Ruby のバージョンのメンテナンス状況を記載します。
```
Comment (sorah) [inline ja/downloads/branches/index.md:7]:
```suggestion
このページでは Ruby の各ブランチについてメンテナンス状況を記載します。
```

File: ja/downloads/branches/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+{: .summary}
+
+より詳細な情報については[リリースページ](../releases/)を見てください。
+
+これらは Ruby のブランチとそのメンテナンス状況の暫定リストです。
```
Comment (sorah) [inline ja/downloads/branches/index.md:12]:
```suggestion
以下は Ruby のブランチとそのメンテナンス状況の暫定リストです。
```

File: ja/downloads/branches/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+EOL の日付等は予定として掲載しています。
+
+Ruby のバージョンごとのブランチのメンテナンス方針は以下のフェーズに別れています:
```
Comment (sorah) [inline ja/downloads/branches/index.md:15]:
```suggestion
Ruby の各ブランチは以下のフェーズに別れてメンテナンスされています:
```

File: ja/downloads/index.md [hunk 5]
```diff
@@ -57,7 +57,11 @@ lang: ja
     毎晩Gitから作成しているtarballです。
     バグやその他の問題があるかもしれません。利用する場合は自己責任でお願いします！
 
-RubyのSubversionやGitリポジトリについての情報は、[リポジトリガイド](/ja/documentation/repository-guide)を参照してください。
+Ruby の Gitリポジトリについての情報は、[リポジトリガイド](/ja/documentation/repository-guide)を参照してください。
```
Comment (sorah) [inline ja/downloads/index.md:60]:
```suggestion
Ruby の Git リポジトリについての情報は、[リポジトリガイド](/ja/documentation/repository-guide)を参照してください。
```

File: ja/downloads/index.md [hunk 6]
```diff
@@ -57,7 +57,11 @@ lang: ja
...
+
+リリースごとの詳しい情報や古いリリース、プレビューなどの情報については[リリースページ][releases]を見てください。
+
+現在メンテナンス状況や Ruby のバージョンごとの情報については[メンテナンスバージョン][branches]のページを見てください。
```
Comment (sorah) [inline ja/downloads/index.md:64]:
```suggestion
各 Ruby バージョンのメンテナンス方針、各ブランチの状況・予定については[ブランチごとのメンテタンス状況][branches]を参照してください。
```

File: ja/downloads/index.md [hunk 7]
```diff
@@ -57,7 +57,11 @@ lang: ja
     毎晩Gitから作成しているtarballです。
     バグやその他の問題があるかもしれません。利用する場合は自己責任でお願いします！
 
-RubyのSubversionやGitリポジトリについての情報は、[リポジトリガイド](/ja/documentation/repository-guide)を参照してください。
+Ruby の Gitリポジトリについての情報は、[リポジトリガイド](/ja/documentation/repository-guide)を参照してください。
+
+リリースごとの詳しい情報や古いリリース、プレビューなどの情報については[リリースページ][releases]を見てください。
```
Comment (sorah) [inline ja/downloads/index.md:62]:
```suggestion
リリースごとの詳しい情報や古いリリース、プレビューなどの情報については[リリースページ][releases]を参照してください。
```

**Review by sorah — COMMENTED** [review]
- Note: In English version, comment states the page is in development so not to translate, if you're happy to start translation then worth to remove the comment?
- Use the term "branches" even in Japanese, to align with English version. I don't think there's no reason to have disparity between languages here.

File: ja/downloads/branches/index.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  これらのブランチは Ruby 開発チームによる修正は種類を問わず行われません。
+
+{% include branches-timeline.html %}
+
+{% for branch in site.data.branches %}
+### Ruby {{ branch.name }}
+
+ステータス: {{ branch.status }}<br>
+リリース: {{ branch.date }}<br>
+通常メンテナンス終了: {% if branch.security_maintenance_date %}{{ branch.security_maintenance_date }}{% else %}TBD{% endif %}<br>
```
Comment (sorah) [inline ja/downloads/branches/index.md:31]:
```suggestion
ノーマルメンテナンス終了: {% if branch.security_maintenance_date %}{{ branch.security_maintenance_date }}{% else %}TBD{% endif %}<br>
```

**Review by sorah — APPROVED** [review]

**Comment by sorah** [comment]
> Note: In English version, comment states the page is in development so not to translate, if you're happy to start translation then worth to remove the comment?

WDYT?

**Comment by hsbt** [comment]
> > Note: In English version, comment states the page is in development so not to translate, if you're happy to start translation then worth to remove the comment?
> 
> WDYT?

I removed development status. Because this page is published while over the 6 years.

**Comment by hsbt** [comment]
@sorah Thank you for your review and suggestions 🚀

**Pull request merged**